docs/quickstart.rst

```diff
@@ -124,7 +124,7 @@ After downloading the dataset and unzipping it in your current working directory
 
 The machine learning workflow is the art of creating *model selection triples*, a combination of features, algorithm, and hyperparameters that uniquely identifies a model fitted on a specific data set. As part of our feature selection, we want to identify features that have a linear relationship with each other, potentially introducing covariance into our model and breaking OLS (guiding us toward removing features or using regularization). We can use the Rank2D_ visualizer to compute Pearson correlations between all pairs of features as follows:
 
-.. _Rank2D: http://www.scikit-yb.org/en/latest/api/yellowbrick.features.html#module-yellowbrick.features.rankd
+.. _Rank2D: https://github.com/DistrictDataLabs/yellowbrick/blob/develop/docs/index.rst#id13
```

Comment:
This line can be removed in favor of the edit above.

Reply:
ok,sure I am correcting it

Reply:
Thank you for being so patient and continuously helping me.I will be contributing regularly,because I like the organisation and I will be participating in gsoc 19

Reply:
@Naba7 - we're happy for your interest in scikit-yb and look forward to GSOC. I'm not seeing any new commits that correct the above edits suggested by @bbengfort -- is it possible that you accidentally pushed those commits to a different branch of your YB fork?